File: src/Carrito.py
class Carrito:
    def __init__(self, producto, cantidad):
        self.__producto = producto
        self.__cantidad = cantidad
        self.comprar(cantidad)

    def get_producto(self):
        return self.__producto

    def subtotal(self):
        return self.__producto.get_precio() * self.__cantidad

    def get_cantidad(self):
        return self.__cantidad

    def comprar(self, cantidad):
        self.__producto.comprar_producto(cantidad)

File: src/Carrito.java
public class Carrito {
    private Producto producto;
    private int cantidad;
    public Carrito(Producto producto,int cantidad) {
        this.producto=producto;
        this.cantidad=cantidad;
        comprar(cantidad);
    }
    public Producto getProducto() {
        return producto;
    }
    public double subtotal(){
        return producto.getPrecio()*cantidad;
    }
    public int getCantidad() {
        return cantidad;
    }
    public void comprar(int cantidad){
        producto.comprarProducto(cantidad);
    }
}
